lib/browser/api/safe-storage.ts

```diff
@@ -0,0 +1,3 @@
+const { safeStorage } = process._linkedBinding('electron_browser_safe_storage');
```

Comment:
And once you have removed the SafeStorage class & are setting the methods directly on the exports dictionary, you can replace this with

```suggestion
const safeStorage = process._linkedBinding('electron_browser_safe_storage');
```